.github/CODEOWNERS

```
@@ -74,14 +22,42 @@
 *.yml @awentzel @janechu @nicholasrice @chrisdholt
 build/ @janechu @nicholasrice @chrisdholt @awentzel
 
-# Figma plugin
-/packages/fast-figma-plugin-msft/ @nicholasrice @bheston @e-vanaubrey @Jeremy-Knudsen
+# Specifications
+/specs/ @EisenbergEffect @chrisdholt @janechu @nicholasrice
 
-# fast-element
-/packages/fast-element/ @EisenbergEffect @chrisdholt @janechu @nicholasrice
+# Sites
+/sites/fast-color-explorer/ @nicholasrice @janechu @bheston
+/sites/fast-component-explorer/ @janechu @nicholasrice
+/sites/website/ @awentzel @chrisdholt @falkicon
 
-# fast-components
-/packages/fast-components/ @EisenbergEffect @chrisdholt @janechu @nicholasrice
+# Package specific owners
 
-# Web Component specs
-/specs/ @EisenbergEffect @chrisdholt @janechu @nicholasrice
\ No newline at end of file
+# React
+/packages/react/fast-components-class-name-contracts-base/ @chrisdholt @janechu @nicholasrice
+/packages/react/fast-components-class-name-contracts-msft/ @chrisdholt @janechu @nicholasrice
+/packages/react/fast-components-foundation-react/ @chrisdholt @janechu @nicholasrice
+/packages/react/fast-components-react-base/ @chrisdholt @janechu @nicholasrice
+/packages/react/fast-components-react-msft/ @chrisdholt @janechu @nicholasrice
+/packages/react/fast-components-styles-msft/ @chrisdholt @janechu @nicholasrice
+/packages/react/fast-jss-manager/ @nicholasrice @janechu @chrisdholt
+/packages/react/fast-jss-manager-react/ @nicholasrice @janechu @chrisdholt
+/packages/react/fast-jss-utilities/ @nicholasrice @janechu @chrisdholt
+/packages/react/fast-layouts-react/ @chrisdholt @nicholasrice
+/packages/react/fast-react-utilities/ @chrisdholt @nicholasrice
+/packages/react/fast-storybook-design-system-addon/ @nicholasrice @chrisdholt @janechu
+/packages/react/fast-storybook-presets/ @nicholasrice @chrisdholt @janechu
+
+# Tooling
+/packages/tooling/fast-figma-plugin-msft/ @nicholasrice @bheston @e-vanaubrey @Jeremy-Knudsen
```

```suggestion
/packages/tooling/fast-figma-plugin-msft/ @nicholasrice @bheston 
```

Those didn't work anyway so we can remove

---

we should likely add one more here just to ensure sign off can take place if you or @bheston are unavailable.

---

Done